_config.yml

```diff
@@ -217,7 +217,9 @@ showSessions: true
 
 # Team Block
 teamBlockTitle: "About us"
-aboutUs: "The OHBM Open Science SIG mission is to advance neuroimaging research by fostering the open sharing of ideas, data, and tools between members of the OHBM community. This will be accomplished by organizing educational events, supporting collaborative initiatives, and providing mentorship to junior researchers. In our first year, the SIG will focus on three contributions to the OHBM annual meeting: 1) organize a Hackathon to precede the annual meeting, 2) introduce the tools of open science by proposing a “Brainhacking 101” educational course, and 3) propose hands-on tutorials and collaborative Open Science projects during the OHBM conference."
+aboutUs: "The OHBM Open Science SIG mission is to advance neuroimaging research by fostering the open sharing of ideas, data, and tools between members of the OHBM community. This will be accomplished by organizing educational events, supporting collaborative initiatives, and providing mentorship to junior researchers.
+As with previous years, our three main contributions to the OHBM annual meetin are: 1) organize a Hackathon to precede the annual meeting, 2) introduce the tools of open science via interactive tutorials as part of the TrainTrack, and 3) facilitate collaborative work on a wide range of open science projects.
```

Comment:
if we find a find a way to render those as a numbered list it would be nice.

But really low priority ATM